### PR TITLE
fix: add/use common tokens + remove multicall2 related

### PR DESCRIPTION
**Pull request by Jeff-CCH**
- [x] common: add common tokens
- [x] common: remove multicall2 related
- [x] test-helpers: remove fixtures and change wstETH faucet mainnet address
- [x] yarn changeset

### CODE REVIEW

File: packages/test-helpers/test/mainnet/faucet.test.ts [hunk 1]
```diff
@@ -19,11 +18,11 @@ describe('mainnet: Test faucet claim', function () {
   snapshotAndRevertEach();
 
   const testCases: { tokenOrAddress: common.TokenOrAddress; amount: string; faucet?: string }[] = [
-    { tokenOrAddress: mainnetTokens.ETH, amount: '1' },
-    { tokenOrAddress: mainnetTokens.WETH, amount: '1' },
+    { tokenOrAddress: common.mainnetTokens.ETH, amount: '1' },
+    { tokenOrAddress: common.mainnetTokens.WETH, amount: '1' },
     { tokenOrAddress: '0xBe9895146f7AF43049ca1c1AE358B0541Ea49704', amount: '1' }, // cbETH
     { tokenOrAddress: '0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0', amount: '1' }, // wstETH
```
Comment (LuPoYi) [inline packages/test-helpers/test/mainnet/faucet.test.ts:24]:
```suggestion
    { tokenOrAddress: common.mainnetTokens.cbETH, amount: '1' },
    { tokenOrAddress: common.mainnetTokens.wstETH, amount: '1' },
```

**Comment by LuPoYi** [comment]
LGTM

File: packages/test-helpers/test/mainnet/faucet.test.ts (hunk 1 above)
Comment (Jeff-CCH) [reply packages/test-helpers/test/mainnet/faucet.test.ts:24]: I think this is intended to test address as function input lol.

Comment (LuPoYi) [reply packages/test-helpers/test/mainnet/faucet.test.ts:24]: oh..you are right XD

**Pull request merged**